README.md

```diff
@@ -85,8 +85,10 @@ To shutdown `mongod` started manually, use the `admin` database and run `db.shut
 $ mongo admin --eval "db.shutdownServer()"
 ```
 
-## Additional Information
+## Additional Information and Problem Reporting
 
 This tap was created using the Homebrew documentation on [How to Create and Maintain a tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).
 
 You can find additional information in the [Homebrew project README](https://github.com/Homebrew/brew#homebrew).
+
+If you're having issues with MongoDB please check out [our community support resources](https://www.mongodb.com/community-support-resources)
```

Comment:
If we are going to disable issues here (and I think I support that, having looked at the issues already filed), I think we should add some language here that points people to the correct place to file issues with the homebrew tap (the SERVER project, presumably). That community support resources page doesn't link to JIRA anywhere.

Reply:
Done.